src/NuGet.Indexing/Extraction/DocumentCreator.cs

```diff
@@ -99,41 +99,24 @@ private static void AddId(Document document, IDictionary<string, string> package
         private static void AddVersion(Document document, IDictionary<string, string> package, List<string> errors)
         {
             string verbatimVersion;
-            if (package.TryGetValue(MetadataConstants.VersionPropertyName, out verbatimVersion))
+            if (package.TryGetValue(MetadataConstants.VerbatimVersionPropertyName, out verbatimVersion))
             {
-                AddField(document, LuceneConstants.VersionPropertyName, verbatimVersion, Field.Index.NOT_ANALYZED);
-            }
-            else
-            {
-                errors.Add($"Required property '{MetadataConstants.VersionPropertyName}' not found.");
-            }
+                AddField(document, LuceneConstants.VerbatimVersionPropertyName, verbatimVersion, Field.Index.NOT_ANALYZED);
 
-            string version;
-            if (!package.TryGetValue(MetadataConstants.NormalizedVersionPropertyName, out version))
-            {
-                if (verbatimVersion != null)
+                NuGetVersion parsedVerbatimVersion;
+                if (NuGetVersion.TryParse(verbatimVersion, out parsedVerbatimVersion))
                 {
-                    NuGetVersion nuGetVersion;
-                    if (NuGetVersion.TryParse(verbatimVersion, out nuGetVersion))
-                    {
-                        version = nuGetVersion.ToNormalizedString();
-                    }
-                    else
-                    {
-                        errors.Add($"Unable to parse '{MetadataConstants.VersionPropertyName}' as NuGetVersion.");
-                    }
+                    AddField(document, LuceneConstants.NormalizedVersionPropertyName, parsedVerbatimVersion.ToNormalizedString(), Field.Index.NOT_ANALYZED);
```

Comment:
Previously, the `NormalizedVersionPropertyName` was marked as `Field.Index.ANALYZED`, now it's `Field.Index.NOT_ANALYZED`. Is that intended? What is the difference between the two?